nemo/collections/asr/models/aed_multitask_models.py

```diff
@@ -1001,7 +926,7 @@ def predict_step(self, batch, batch_idx=0, dataloader_idx=0, has_processed_signa
             signal = None
             signal_len = None
 
-        transf_log_probs, encoded_len, enc_states, enc_mask = self.forward(
+        _, _, enc_states, enc_mask = self.forward(
```

Comment:
same here, lets keep descriptions instead of _, even though we don't use them.

Reply:
Done, this is out of style with the rest of module though.